Formula/h/hdf5.rb

```diff
@@ -43,32 +42,25 @@ def install
               "settingsdir=$(libdir)",
               "settingsdir=#{pkgshare}"
 
-    system "autoreconf", "--force", "--install", "--verbose"
-
-    args = %W[
-      --disable-dependency-tracking
-      --disable-silent-rules
-      --enable-build-mode=production
-      --enable-fortran
-      --enable-cxx
-      --prefix=#{prefix}
-      --with-szlib=#{Formula["libaec"].opt_prefix}
-    ]
-    args << "--with-zlib=#{Formula["zlib"].opt_prefix}" if OS.linux?
-
-    system "./configure", *args
+    system "cmake", 
+      "-S", ".", 
+      "-B", "build", 
+      "-DHDF5_BUILD_FORTRAN:BOOL=ON", 
+      "-DHDF5_BUILD_CPP_LIB:BOOL=ON", 
+      *std_cmake_args
```

Comment:
Sorry! Trailing whitespaces. I would do those changes myself but I don't think I have write access.
Let me know if you want me to create a new PR by myself to avoid this in the future (although I think we might be done soon)

```suggestion
    system "cmake",
      "-S", ".",
      "-B", "build",
      "-DHDF5_BUILD_FORTRAN:BOOL=ON",
      "-DHDF5_BUILD_CPP_LIB:BOOL=ON",
      *std_cmake_args
```

Reply:
Yeah, if this one fails just make a new PR.